test/test_ccf.py

```diff
@@ -141,3 +141,39 @@ def submit(**kwargs):
         submit(alg="ES256", kty="ec", ec_curve="P-256")
         submit(alg="ES384", kty="ec", ec_curve="P-384")
         submit(alg="PS256", kty="rsa")
+
+
+def test_accepted_did_issuers(tmp_path: Path):
+    def not_allowed(f):
+        with pytest.raises(ServiceError, match="InvalidInput: Unsupported did issuer in protected header"):
+            f()
+
+    with SCITTFixture(tmp_path) as fixture:
+
+        def submit(**kwargs):
+            """Sign and submit the claims with a new identity"""
+            identity = fixture.did_web_server.create_identity(**kwargs)
+            claims = crypto.sign_json_claimset(identity, {"foo": "bar"})
+            fixture.client.submit_claim(claims)
+
+        # Start with a configuration with no accepted issuers.
+        # The service should reject anything we submit to it.
+        fixture.configure_service({"policy": {"accepted_did_issuers": []}})
+        not_allowed(lambda: submit(alg="ES256", kty="ec", ec_curve="P-256"))
+
+        # Add just one issuers to the policy. Claims signed with this
+        # issuers are accepted.
+        identity = fixture.did_web_server.create_identity(alg="ES256", kty="ec", ec_curve="P-256")
```

Comment:
Using an explicit `alg`/`kty`/`ec_curve` here makes it seem like it is relevant to the current test, when actually we don't really care in this context. Removing those reduces the noise and helps highlight what actually matters.

```suggestion
        identity = fixture.did_web_server.create_identity()
```